.pipelines/cni/pipeline.yaml

```diff
@@ -111,16 +109,31 @@ stages:
       nodeCount: ${CILIUM_NODE_COUNT}
       vmSize: "Standard_DS4_v2"
 
-  - template: singletenancy/windows-cni-load-test-template.yaml
+  - template: singletenancy/cniv1-template.yaml
     parameters:
-      name: win_cniv1
-      clusterType: "windows-cniv1-up"
-      clusterName: "win-cniv1"
+      name: win22_cniv1
+      clusterType: windows-cniv1-up
+      clusterName: win22-cniv1
       nodeCount: ${WINDOWS_NODE_COUNT}
       vmSize: Standard_DS4_v2
-      windowsVMSize: ${WINDOWS_VM_SKU}
-      arch: amd64
+      windowsVMSize: Standard_DS4_v2

```

Comment:
Can we parameterize this ? 
As windows scale up is also parameterized we sometimes need to upgrade the vm size based on that

Reply:
parameterized all related fields.